experimental/split_grpc/server/src/main.rs

```diff
@@ -0,0 +1,84 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use futures_core::Stream;
+use std::pin::Pin;
+use tonic::{transport::Server, Request, Response, Status, Streaming};
+
+pub mod proto {
+    include!("./proto/oak.examples.hello_world.rs");
+}
+
+use proto::hello_world_server::{HelloWorld, HelloWorldServer};
+use proto::{HelloRequest, HelloResponse};
+
+#[derive(Debug, Default)]
+pub struct HelloHandler {}
```

Comment:
(optional) Would `HelloWorldServer` be a more accurate name for this?

Reply:
Renamed to HelloWorldHandler. There is already a HelloWorldServer that is generated as part of compiling the proto. This provides an implementation of the HelloWorld trait to handle the calls. Alternatively perhaps HelloWorldProvider?